.env.example

```
@@ -2,8 +2,8 @@ CLICKHOUSE_URL="http://default:@localhost:8123/databuddy_analytics"
 DATABASE_URL="postgres://databuddy:databuddy_dev_password@localhost:5432/databuddy"
 REDIS_URL="redis://localhost:6379"
 
-# Openrouter API key, only needed for the assistant
-AI_API_KEY=""
+# AI Gatway Vercel API key, only needed for the assistant
+AI_GATEWAY_API_KEY=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)